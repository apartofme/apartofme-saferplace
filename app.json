{
  "name": "SaferPlace",
  "displayName": "Safer Place"
}
