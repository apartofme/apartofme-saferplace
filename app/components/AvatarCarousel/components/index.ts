export * from './AvatarCarouselItem';
export * from './ProgressBarItem';
