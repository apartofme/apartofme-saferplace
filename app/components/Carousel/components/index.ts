export * from './SubtitleImage/SubtitleImage';
export * from './ImageTitleSubTitle';
export * from './ImageSubtitle';
export * from './OnlyImage';
export * from './ProgressBarItem';
