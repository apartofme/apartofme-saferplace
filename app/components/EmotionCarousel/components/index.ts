export * from './EmotionCarouselItem';
export * from './ProgressBarItem';
