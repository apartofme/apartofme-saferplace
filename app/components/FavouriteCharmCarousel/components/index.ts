export * from './FavouriteCharmCarouselItem';
export * from './ProgressBarItem';
