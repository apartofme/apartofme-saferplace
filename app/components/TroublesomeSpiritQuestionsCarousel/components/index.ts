export * from './TroublesomeSpiritQuestionsCarouselItem';
export * from './ProgressBarItem';
