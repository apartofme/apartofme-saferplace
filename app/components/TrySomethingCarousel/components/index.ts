export * from './TrySomethingCarouselItem';
export * from './ProgressBarItem';
