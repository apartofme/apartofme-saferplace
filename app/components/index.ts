export * from './ExtendedButton';
export * from './ExtendedText';
export * from './ExtendedTextInput';
