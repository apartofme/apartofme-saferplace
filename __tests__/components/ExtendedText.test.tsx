import 'react-native';
import React from 'react';
import testRenderer from 'react-test-renderer';

import { ExtendedText } from '../../app/components';

test('UI: ExtendedText should match snapshot', () => {
  // No preset prop
  const actualExtendedText = testRenderer
    .create(<ExtendedText>test text</ExtendedText>)
    .toJSON();
  expect(actualExtendedText).toMatchSnapshot();

  // Preset "title" prop
  const actualExtendedTitleText = testRenderer
    .create(<ExtendedText preset="title">test title text</ExtendedText>)
    .toJSON();
  expect(actualExtendedTitleText).toMatchSnapshot();

  // Preset "secondary" prop
  const actualExtendedSecondaryText = testRenderer
    .create(<ExtendedText preset="secondary">test secondary text</ExtendedText>)
    .toJSON();
  expect(actualExtendedSecondaryText).toMatchSnapshot();
});
