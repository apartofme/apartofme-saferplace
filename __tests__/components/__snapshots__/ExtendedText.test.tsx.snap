// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI: ExtendedText should match snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#333",
        "fontSize": 15,
      },
      undefined,
    ]
  }
>
  test text
</Text>
`;

exports[`UI: ExtendedText should match snapshot 2`] = `
<Text
  style={
    Array [
      Object {
        "color": "#333",
        "fontSize": 24,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
>
  test title text
</Text>
`;

exports[`UI: ExtendedText should match snapshot 3`] = `
<Text
  style={
    Array [
      Object {
        "color": "#333",
        "fontSize": 11,
      },
      undefined,
    ]
  }
>
  test secondary text
</Text>
`;
